### PR TITLE
layouts: webauthn and management

### PR DESCRIPTION
**Pull request by mmilata**
This should be the final major PR in the _layouts_ series as pretty much all UI now works using layouts.

Previously all TT layouts were in a single file (`trezor/ui/layouts/tt.py`), this PR splits them into multiple files but does not add the "import magic" (i.e. `trezor/ui/layouts/__init__.py`) because it's not clear it will be needed (or if we're gonna RIIR the layouts).

Regarding memory this branch passes device tests (w/ `--random-order-bucket=none`) with minimal `--heap-size` set to 459K while master needs 461K. Might be because this branch diverged from master quite some time ago, needs rechecking.

TODO after #1538 approved: go over changes once more, consider test coverage, post UI diff

### CODE REVIEW

**Comment by matejcik** [comment]
I rebased this and made some minor tweaks (most notably 073b412) in a separate branch: https://github.com/trezor/trezor-firmware/compare/matejcik/layouts-4?expand=1

I still want to do a detailed review, but it looks good overall.

@mmilata want to look it over if the rebase is right from your POV? or should I forcepush directly into your branch?

**Comment by mmilata** [comment]
Rebase ack, please push with great force.

File: core/src/apps/management/change_wipe_code.py [hunk 1]
```diff
@@ -56,27 +53,41 @@ async def change_wipe_code(ctx: wire.Context, msg: ChangeWipeCode) -> Success:
     return Success(message=msg_wire)
 
 
-def _require_confirm_action(
+async def _require_confirm_action(
```
Comment (matejcik) [inline core/src/apps/management/change_wipe_code.py:56]: any particular reason you made this `async` (as opposed to using return type `Awaitable[None]`)?

**Comment by matejcik** [comment]
I made some fixups and a minor UI change, please review. Also I'd like to fixup the `async` thing unless there's reason for it.

Otherwise LGTM. Not sure if this requires a changelog entry?

**Comment by matejcik** [comment]
i'd like to wait with merging until QA has had a go at the U2F changes

**Comment by bosomt** [comment]
**QA OK**

File: core/src/apps/management/change_wipe_code.py (hunk 1 above)
Comment (mmilata) [reply core/src/apps/management/change_wipe_code.py:56]: no reason, feel free to change it

**Review by mmilata — COMMENTED** [review]
Fixups ACK. Not sure about the changelog as there are no functional and very little visual changes?

**Comment by matejcik** [comment]
Thanks. Squashed, rebased, added changelog (because previous layout changes also have it)

**Comment by bosomt** [comment]
already tested, moving to approved

**Pull request merged**